### ISSUE TITLE
Pupnet pre-requisites for building RPM packages

### ISSUE THREAD

**Issue by stuarthillary** [kuiperzone/PupNet-Deploy#7]
As noted in the README, I installed rpmdevtools as a dependency of Pupnet when creating RPM packages. After doing this I tried to create an RPM package for the PupNet-Helloworld project. 

The command

`pupnet -k rpm`

failed, with a message 

`desktop-file-validate: command not found`

I fixed this by installing the desktop-file-utils package. The RPM package was then successfully created. I am guessing the README just needs to include this additional dependency.

**Comment by kuiperzone** [comment]
Certainly I can add it to the read me. But it doesn't sound like a problem with PupNet as it doesn't call `desktop-file-validate` directly. Rather rpmbuild is calling it and it should therefore install it if it needs it.

What distro are you using?

**Comment by stuarthillary** [comment]
I am using RHEL 8.7. The package `rpmdevtools` doesn't have `desktop-file-utils` as an explicit package dependency

```
sudo yum repoquery --deplist rpmdevtools
Updating Subscription Management repositories.
Last metadata expiration check: 21:10:00 ago on Thu 13 Apr 2023 12:06:53 PM BST.
package: rpmdevtools-8.10-7.el8.noarch
  dependency: /bin/bash
   provider: bash-4.4.20-4.el8_6.x86_64
   provider: bash-4.4.20-4.el8_6.x86_64
  dependency: /bin/sh
   provider: bash-4.4.20-4.el8_6.x86_64
   provider: bash-4.4.20-4.el8_6.x86_64
  dependency: /usr/bin/perl
   provider: perl-interpreter-4:5.26.3-421.el8.x86_64
   provider: perl-interpreter-4:5.26.3-421.el8.x86_64
  dependency: /usr/libexec/platform-python
   provider: platform-python-3.6.8-48.el8_7.1.i686
   provider: platform-python-3.6.8-48.el8_7.1.x86_64
   provider: platform-python-3.6.8-48.el8_7.1.x86_64
   provider: platform-python-3.6.8-48.el8_7.1.i686
  dependency: curl
   provider: curl-7.61.1-25.el8_7.3.x86_64
   provider: curl-7.61.1-25.el8_7.3.x86_64
  dependency: diffutils
   provider: diffutils-3.6-6.el8.x86_64
   provider: diffutils-3.6-6.el8.x86_64
  dependency: emacs-filesystem
   provider: emacs-filesystem-1:26.1-7.el8.noarch
   provider: emacs-filesystem-1:26.1-7.el8.noarch
  dependency: file
   provider: file-5.33-21.el8.x86_64
   provider: file-5.33-21.el8.x86_64
  dependency: findutils
   provider: findutils-1:4.6.0-20.el8.x86_64
   provider: findutils-1:4.6.0-20.el8.x86_64
  dependency: gawk
   provider: gawk-4.2.1-4.el8.x86_64
   provider: gawk-4.2.1-4.el8.x86_64
  dependency: grep
   provider: grep-3.1-6.el8.x86_64
   provider: grep-3.1-6.el8.x86_64
  dependency: perl(File::Spec)
   provider: perl-PathTools-3.74-1.el8.x86_64
   provider: perl-PathTools-3.74-1.el8.x86_64
  dependency: perl(File::Temp)
   provider: perl-File-Temp-0.230.600-1.el8.noarch
   provider: perl-File-Temp-0.230.600-1.el8.noarch
  dependency: perl(FileHandle)
   provider: perl-interpreter-4:5.26.3-421.el8.x86_64
   provider: perl-interpreter-4:5.26.3-421.el8.x86_64
  dependency: perl(Getopt::Long)
   provider: perl-Getopt-Long-1:2.50-4.el8.noarch
   provider: perl-Getopt-Long-1:2.50-4.el8.noarch
  dependency: perl(strict)
   provider: perl-libs-4:5.26.3-421.el8.i686
   provider: perl-libs-4:5.26.3-421.el8.x86_64
   provider: perl-libs-4:5.26.3-421.el8.i686
   provider: perl-libs-4:5.26.3-421.el8.x86_64
  dependency: python3-rpm
   provider: python3-rpm-4.14.3-24.el8_7.x86_64
   provider: python3-rpm-4.14.3-24.el8_7.x86_64
  dependency: rpm-build >= 4.4.2.3
   provider: rpm-build-4.14.3-24.el8_7.x86_64
   provider: rpm-build-4.14.3-24.el8_7.x86_64
  dependency: sed
   provider: sed-4.5-5.el8.x86_64
   provider: sed-4.5-5.el8.x86_64
```

**Comment by kuiperzone** [comment]
This should be fixed in 1.2.2.  Added 'BuildRequires: desktop-file-utils' to RPM spec file
